src/plugins/template/tests/functional/op_reference/identity.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/identity.hpp"
+
+#include "base_reference_test.hpp"
+#include "gtest/gtest.h"
+#include "openvino/op/parameter.hpp"
+
+namespace {
+struct IdentityParams {
```

Comment:
Use tuple instead e.g.
```suggestion
using IdentityParams = std::tuple<reference_tests::Tensor, std::string>;
```
?

Reply:
Applied!